recipes-kernel/linux/files/iot2050_defconfig_base

```diff
@@ -444,7 +437,6 @@ CONFIG_REGULATOR_GPIO=y
 CONFIG_REGULATOR_HI6421V530=y
 CONFIG_REGULATOR_MAX77620=y
 CONFIG_REGULATOR_PALMAS=y
-CONFIG_REGULATOR_PWM=y
```

Comment:
Interesting that this automatically dropped off. But checking our DT, I don't see we ever needed it.